### DISCUSSION TITLE
How to run /login command on server join

### DISCUSSION THREAD

**Discussion by SoulCatcher12** [ORelio/Minecraft-Console-Client#2000] [Q&A] [answered]
hello guys can anyone tell how to make bot execute commands like /register (password) (password) because cracked servers use this format also can anyone tell how to make bot join party and execute command like /party join (Username)

also when bot is registered and want to login 2nd time how use /login command

**Comment by ORelio** [comment, marked as answer]
Use [TaskScheduler](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#using-the-script-scheduler--task-scheduler) with `action=send /yourcommand` on server join.
